### ISSUE TITLE
[BUG] Lists in Rich Text Fields not displaying properly in delivered template (GeneralSLDS)

### ISSUE THREAD

**Issue by Scarrien** [SFDO-Community-Sprints/Summit-Events-App#409]
When viewing a registration page that use the delivered "GeneralSLDS" template, what is entered in the "Event Description" field on the Summit Event object should match what is displayed. However this is not currently the case, as features like bullet points and numbered lists in the Event Description field aren't displaying the same way on the registration page.

Examples of this are

- Bullet points are displaying to the left, with the text itself being aligned with the text above and below. It should be that the bullet points themselves should line up with the surrounding text, with the text indented to match.
1. Numbered lists are not displaying as a list at all, instead being just displaying the individual points.

In talking with Thad, it appears the issue is a difference between the value of the rich text field and the stylizing of Salesforce's SLDS. I believe the solution is to either add CSS to the delivered template or see if the [apex:outputText](https://developer.salesforce.com/forums/?id=9062I000000IHjCQAW) component can be used to account for these differences.

![Event Description field rich text bulleted list](https://user-images.githubusercontent.com/50189636/170563391-113b2092-69c5-48a0-be82-eab358b9cb35.png)
![Registration page with General SLDS template not displaying rich test bulleted list](https://user-images.githubusercontent.com/50189636/170563385-0b293d29-9d4d-422a-b76b-6e8b5bc99cb9.png)

I have the code to fix these two examples, but they should be included in the template by default instead of relying on the users to fix this.
```
.slds-scope ul {
	list-style: disc !important;
	margin: initial !important;
	padding: 0px 0px 0px 30px !important;
}

.slds-scope ol {
	list-style: decimal;
	margin: inherit;
	padding: 0px 0px 0px 30px;
}
```

**Comment by Scarrien** [comment]
These were only the two problems I was solving, other rich text editor features such as indents haven't been tested so they need to be checked to see if they work with SLDS. These may require additional edits to our template.

**Comment by tcdahlberg** [comment]
Included in beta release 0.28.0.1

**Comment by tcdahlberg** [comment]
Included in production release 0.28.0.2
